src/operators/convert.ts

```diff
@@ -210,14 +212,17 @@ export class ConvertOperator implements Operator {
         : !Number.isNaN((newValue as Date).getTime());
     }
 
-    // log warning and reset to the original value
+    // Note `debug` level is used because `enumerate` may always be done `beforeDestination`, which could generate
+    // a lot of false positives
+    // log debug and reset to the original value
     if (!verified) {
-      Logger.getInstance().warn(LogMessageType.OperatorError, {
+      newMap[property] = oldValue; // eslint-disable-line no-param-reassign
```

Comment:
Note I moved the assignment above the Logger in case something in it could've prevented that from occurring.  (I could've also try catched the Logger, but figured this was sufficient with less code.)